lib/dal/src/workspace_snapshot/node_weight.rs

```diff
@@ -677,7 +677,7 @@ impl NodeWeight {
         Ok(NodeWeight::FuncArgument(FuncArgumentNodeWeight::new(
             change_set,
             func_arg_id,
-            ContentAddress::Func(content_hash),
```

Comment:
This was the cause of the bug.

Reply:
Yes, this massive diff for basically one line that caused `FuncArgument` type switching to not work. LOL.